.github/workflows/ci.yml

```diff
@@ -60,7 +72,18 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
 
-      - run: go build -v .
+      # xcode required until github macos runner is 11.x based
+      - uses: maxim-lobanov/setup-xcode@v1
+        if: matrix.target.goos == 'darwin' && matrix.target.goarch == 'arm64'
+        with:
+          xcode-version: latest
+
+      - if: matrix.target.goos == 'darwin' && matrix.target.goarch == 'arm64'
+        run: |
+          echo SDKROOT=$(xcrun --sdk macosx --show-sdk-path) >> $GITHUB_ENV
```

Comment:
How come we don't need this for `amd64`? I'm guessing this is some quirk of how GitHub set up their macOS builders?

Reply:
Pretty much, the default GitHub ubuntu/macos runners come with Xcode already installed but it's an older version before darwin/arm64

Reply:
Fair enough, will just need to keep an eye out for when they update their builders. I want to land a few cleanups before the next release, and we'll be able to push something out (with Windows and M1 binaries) soon. Thanks again!